packages/cli/src/commands/generate/function/function.js

```diff
@@ -48,19 +48,65 @@ export const builder = (yargs) => {
       description: 'Name of the Function',
       type: 'string',
     })
+    .option('force', {
+      alias: 'f',
+      default: false,
+      description: 'Overwrite existing files',
+      type: 'boolean',
+    })
     .epilogue(
       `Also see the ${terminalLink(
         'Redwood CLI Reference',
         'https://redwoodjs.com/reference/command-line-interface#generate-function'
       )}`
     )
+
   // Add default options, includes '--typescript', '--javascript', '--force', ...
   Object.entries(yargsDefaults).forEach(([option, config]) => {
     yargs.option(option, config)
   })
 }
 
-export const { command, handler } = createYargsForComponentGeneration({
-  componentName: 'function',
-  filesFn: files,
-})
+export const handler = async ({ name, force }) => {
+  const tasks = new Listr(
+    [
+      {
+        title: `Generating function files...`,
+        task: async () => {
+          const f = await files({ name })
+          return writeFilesTask(f, { overwriteExisting: force })
+        },
+      },
+    ],
+    { collapse: false, exitOnError: true }
+  )
+
+  try {
+    await tasks.run()
+
+    console.info('')
+
+    console.info(c.warning('⚠ Important:'))
+    console.info('')
+    console.info(
+      c.bold(
+        'When deployed, a custom serverless function is an open API endpoint and is your responsibility to secure appropriately.'
+      )
+    )
+
+    console.info('')
+
+    console.info(
+      c.info(
+        `Please consult the ${terminalLink(
+          'Serverless Function Considerations',
+          'https://redwoodjs.com/docs/serverless-functions#security-considerations'
+        )} in the RedwoodJS documentation for more information.`
```

Comment:
```suggestion
        `Please consult the following documentation for information:\n${terminalLink(
          'Serverless Function Considerations',
          'https://redwoodjs.com/docs/serverless-functions#security-considerations'
        )}`
```

The link was getting a bit lost in the text for me. What do you think about this? (Also, the 'info' color is super light in my console... not sure there's much to do about that in this PR.)

<img width="773" alt="Screen Shot 2021-04-05 at 11 13 28 PM" src="https://user-images.githubusercontent.com/2951/113666744-91af1a80-9664-11eb-84c8-e70fc127ec95.png">

Reply:
Added line break. Done.

> light in my console... not sure there's much to do about that in this PR.

That's VSCode's gray and what "info" is set to in colors.

But am changing to not use info and looks better:

![image](https://user-images.githubusercontent.com/1051633/113704114-bbd6fb80-96a9-11eb-9bdc-1b623896ff82.png)